CHANGELOG.md

```diff
@@ -8,18 +8,41 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Support for `idempotencyKey` for `POST` methods by @aydrian & @rileylnapier
+- Support for [Lists API](https://docs.courier.com/reference/lists-api) by @aydrian
+  - `POST /send/list` via `client.sendListOrPattern(params, config)`
```

Comment:
we are now starting to think about pattern sending as more of a business feature. can we default to `sendList` everywhere? i know that's a change from how we were thinking about it w/ topics, but the requirement to expand past 4 pattern parts changed the thinking.

Reply:
another suggest was organizing more like this with reducing redundancy with verbage:
- `client.lists.send()`

> strange one, but `client.profiles.recipient.lists.list(recipientId)` seems odd, despite being consistent
- `GET /profiles/{recipient_id}/lists` via `client.profiles.getRecipientLists(recipientId)`

> we like the standard of `.list` despite being a little strange in this one use case
- `GET /lists` via `client.lists.list(params)`  || `client.lists.getItems(params)` 

- `GET /lists/{list_id}` via `client.lists.get(listId)`
- `PUT /lists/{list_id}` via `client.lists.replace(listId, {...})`
- `DELETE /lists/{list_id}` via `client.lists.delete(listId)`
- `PUT /lists/{list_id}/restore` via `client.lists.restore(listId)`
- `GET /lists/{list_id}/subscriptions` via `client.lists.getSubscriptions(listId)`
- `POST /lists/{list_id}/subscriptions` via `client.lists.bulkSubscribe(listId, [recipientId], config)`
- `PUT /lists/{list_id}/subscriptions/{recipient_id}` via `client.lists.subscribe(listId, recipientId)`
- `DELETE /lists/{list_id}/subscriptions/{recipient_id}` via `client.lists.unsubscribe(listId, recipientId)`

the intent here is to create some standards around nesting of our api objects. since subscriptions can't live on their own with a list, the functionality to support subscriptions is up-leveled to the `lists` object (as opposed to client.lists.subscriptions.delete`. generic pattern is `client.feature.action`.

Reply:
once we all agree on the structure, we need a tix to go back and review our other implementations here

Reply:
also see comment right below this talking about `create + update` instead of `replace`

Reply:
So looks like you found the one place I didn't change it to sendList

Reply:
Ah. Yes. I see what you're saying. I didn't know if I should leave the redundancy or try to take it out. Remember I moved these functions around and I'm trying to keep it backwards compatible. I can probably remove it. I think that's what I've done in other non courier libraries.